README.md

```diff
@@ -47,6 +47,11 @@ Encode a transaction object into a hex-string. Note that encode filters out fiel
   * If other fields (in the future) must to support X-addresses with tags, this library will need to be updated.
   * When decoding rippled binary, the output will always output classic address + tag, with no X-addresses. X-address support only applies when encoding to binary.
 
+#### Encoding Currency Codes
+  * ripple-binary-codec allows for any 3 character ASCII string to be encoded into a [currency code](https://xrpl.org/currency-formats.html#currency-codes).
+  * ripple-binary-codec will decode currencies into that are valid ISO 4217 codes.
```

Comment:
```suggestion
  * When _decoding_, if a currency code is three uppercase letters or numbers (`/^[A-Z0-9]{3}$/`), then it will be decoded into that string. For example,`0000000000000000000000004142430000000000` decodes as `ABC`.
```